src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
Gotcha. I do most of my work on a Mac at the moment, so I'll keep this in mind if doing any more work on this - time to break out the VM 😉